app/models/person_with_remote_fields.rb

```diff
@@ -7,12 +7,11 @@ class PersonWithRemoteFields < Person
 
   attr_accessor *REMOTE_FIELDS
 
-
   def save(args = {})
     if valid?
       update_remote
     end
-    super # will try to save location, but won't complain if it can't
```

Comment:
since we're using `validates_associated :location`, person won't save if location is invalid.

The error message will be "location is invalid", not "zip code can't be blank", etc. I figure this will be good enough for now.